site/pages/provisional-en.hbs

```diff
@@ -201,18 +201,30 @@
 		<td>v8.0.1</td>
 		<td></td>
 	</tr>
-		<tr>
-			<th>CSS <code>.gc-chckbxrdio</code></th>
-			<td>
-				<p>Adopt the new large input touch targets for checkboxes and radio-buttons</p>
-				<ul>
-					<li><a href="component/gc-chckbxrdio-en.html">Checkboxes and radio buttons - Canada.ca design pattern</a></li>
-				</ul>
-			</td>
-			<td>v8.0.1</td>
-			<td></td>
-		</tr>
-	</tbody>
+	<tr>
+		<th>CSS <code>.gc-chckbxrdio</code></th>
+		<td>
+			<p>Adopt the new large input touch targets for checkboxes and radio-buttons</p>
+			<ul>
+				<li><a href="component/gc-chckbxrdio-en.html">Checkboxes and radio buttons - Canada.ca design pattern</a></li>
+			</ul>
+		</td>
+		<td>v8.0.1</td>
+		<td></td>
+	</tr>
+	<tr>
+		<th>CSS <code>.followus</code></th>
+		<td>Adopt the new social media styles</td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<th>CSS <code>.followus-vertical</code></th>
+		<td>Styles used for the vertical social media block</td>
```

Comment:
Add link to it's working example which are bellow.

Reply:
Added